pitch.md

```diff
@@ -57,9 +73,12 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think that a significant portion of my time would be spent with language aspects, but I do recognize the pretty significant systems portion of actually building the constraint solver. One thing that I think would push me further into the language design decisions area would be the creation of the GUI which would essentially double the amount of decisions I have to make because, as we all know, GUIs are DSLs also! I think I would shoot for 50/50.
```

Comment:
If/when you do start aiming towards a GUI, a lot of time might be spent trying to find the framework to do it in. To avoid that, you might constrain yourself by just doing a really terrible-looking super simple HTML form or whatever else seems easiest.

(You might even be able to get away by using a pre-made form builder tool, or even Google forms, depending.)